src/five/localsitemanager/localsitemanager.txt

```diff
@@ -40,11 +37,11 @@ setting up a test utility.
 
     >>> class ITestUtility(interface.Interface): pass
     >>> class TestUtility(object):
-    ...     interface.implements(ITestUtility)
     ...     def __init__(self, id):
     ...         self.id = id
     ...     def __repr__(self):
     ...         return '<Utility %s "%s">' % (self.__class__.__name__, self.id)
+    >>> TestUtility = interface.implementer(ITestUtility)(TestUtility)
```

Comment:
This does not look very nice. Is there a reason you could not use a class decorator?
(Same for the classes below.)